Video.js

```diff
@@ -180,7 +180,7 @@ export default class Video extends Component {
         uri,
         isNetwork,
         isAsset,
-        type: source.type,
+        type: source.type || 'mp4',
```

Comment:
The exoplayer implementation can infer the type of video, but can be overridden by the `type` field.
By adding a default value the player doesn't infer the type and just assumes it is mp4, and playback fails if it isn't mp4.

Could we make this an empty string instead?